ansible/playbooks/adhoc/hotfix/roles/update_master/tasks/main.yml

```diff
@@ -36,6 +34,7 @@
 - name: Set downgrade_successful=True if downgrade succeeded
   set_fact: 
     downgrade_successful: True
+    upgrade_successful: True
```

Comment:
"upgrade_successful" is confusing in the downgrade case.

Consider calling it something like "wanted_version_installed" or "operation_successful". Something that can cover both update and downgrade.